tests/Type/data/eloquent-builder-l10-27.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace ModelRelations;
```

Comment:
```suggestion
namespace EloquentBuilderL1027;
```

or something like that